src/test/java/picard/util/LiftoverVcfTest.java

```diff
@@ -1169,4 +1173,53 @@ public void testLiftOverNoCallAndSymbolic(final LiftOver liftOver, final Variant
             Assert.assertEquals(resultAlleles, result.getAttributeAsStringList(LiftoverVcf.ORIGINAL_ALLELES, null));
         }
     }
+
+    @Test()
+    public void testNoDictionary() throws IOException {
+        final Path liftOutput = Files.createTempFile("tmpouput", ".vcf");
```

Comment:
with all these deleteOnExit, I wonder if it's cleaner to have a 
try{
....
} finally {
 TestUtil.recursiveDelete(...);
}

Reply:
maybe not....nevermind.